tests/network/localnet/migration_stuntime/conftest.py

```
@@ -0,0 +1,123 @@
+        client_vm.wait_for_agent_connected()
+        # Clear node selector to allow migration to any node
+        client_vm.update_template_node_selector(node_selector=None)
+        yield client_vm
+
+
+@pytest.fixture()
+def stuntime_active_ping(
+    request: pytest.FixtureRequest,
+    localnet_stuntime_server_vm: BaseVirtualMachine,
+    localnet_stuntime_client_vm: BaseVirtualMachine,
+) -> Generator[libstuntime.ContinuousPing, None, None]:
```

Ruff (0.15.10)

[warning] Unnecessary default type arguments

Remove default type arguments

(UP043)